cli/azd/extensions/azure.coding-agent/version.txt

```diff
@@ -0,0 +1 @@
+0.0.1-beta.1

```

Comment:
Do we have any expectation/rule for a syntax here? @wbreza ?

I see we currently have a mix of `x.x.x-preview`, `x.x.x` and `x.x.x-beta.x`  .  Is the Extension framework able to parse versions beyond semver (x.x.x) ?

Reply:
Made it simpler.